docs/source/index.rst

```diff
@@ -1,85 +1,77 @@
-.. PyNIDM documentation master file, created by
-   sphinx-quickstart on Mon Aug 13 11:52:15 2018.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
-
 PyNIDM: Neuroimaging Data Model in Python
 ##########################################
-A Python library to manipulate the [Neuroimaging Data Model](http://nidm.nidash.org).
-
-
-.. toctree::
-   :maxdepth: 2
-   :caption: Contents:
-
-
-Indices and tables
-==================
-
-* :ref:`genindex`
-* :ref:`modindex`
-* :ref:`search`
 
+A Python library to manipulate the `Neuroimaging Data Model
+<http://nidm.nidash.org>`_.
 
 |PyNIDM Testing| |Docs|
 
-.. contents::
-.. section-numbering::
+.. |PyNIDM Testing| image:: https://github.com/incf-nidash/PyNIDM/actions/workflows/pythontest.yml/badge.svg
+    :target: https://github.com/incf-nidash/PyNIDM/actions/workflows/pythontest.yml
+    :alt: Status of PyNIDM Testing
 
+.. |Docs| image:: https://readthedocs.org/projects/pynidm/badge/?version=latest&style=plastic
+    :target: https://pynidm.readthedocs.io/en/latest/
+    :alt: ReadTheDocs Documentation of master branch
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
 
 Dependencies
 ============
-* Git-annex <https://git-annex.branchable.com/install/>
-* Graphviz <http://graphviz.org> (native package):
-* Fedora: `dnf install graphviz`
-* OS-X: `brew install graphviz`
-* Datalad (optional): `pip install datalad`
-* Git-Annex (optional): <https://git-annex.branchable.com/>
+* `Git-annex <https://git-annex.branchable.com/install/>`_
+* `Graphviz <http://graphviz.org>`_ (native package):
+
+  * Fedora: `dnf install graphviz`
+  * OS-X: `brew install graphviz`
```

Comment:
oh, this is all just duplicating the README.rst, eventually we should "solve" that one way or another.

Reply:
* The files are actually not duplicates; see #358 
* Is the documentation hosted in a rendered form anywhere?  If not, I don't think we should be pointing people to it, as GitHub doesn't render it all properly.